package.json

```diff
@@ -6,6 +6,7 @@
     "bin": "dist/app.js",
     "scripts": {
         "start": "tsc && node dist/app.js",
+        "test": "jest",
```

Comment:
I think some configuration is needed. You could check other jest config files e.g. in the Codecharta repo (visualization). 

Maybe Running jest without and config does not work because we are not sticking to the jest standards (test file names, default pathes for test resources, etc. ) or simply a Minimum configuraion is needed.

Reply:
AFAIK the config file has been moved to src/parser/config/nodeTypesConfig.json
For sure it was forgotten to adjust the test. 

Hmm, what is types uses for?

Reply:
I suggest to switch to the built-in Node.js test runner or mocha.

Jest is slow and does pretty bad things internally that features are not working as expected (e.g., `instanceof` being broken, it manipulates assertion libraries output and handling and more).

Reply:
> AFAIK the config file has been moved to src/parser/config/nodeTypesConfig.json For sure it was forgotten to adjust the test.
> 
> Hmm, what is types uses for?

The issue of the wrong path is not within the test cases, but within the application logic (src/parser/CouplingCalculator.ts). 

However, I have changed the path to the file you suggested and that fixed the issue. The test cases still fail, but this is because the generated JSON contains the absolute file paths to the parsed files on several places, which has of course a different prefix compared to `C:\Users\CedrikB\projects\node-parser"` in my case. I guess this is unavoidable as long as the absolute file path is included in the JSON and the test case compares the complete JSON output. 
As the rest of the output is identical, I guess that the application code runs fine now.

Regarding the types field, as this is always empty, I don't know what it is used for. I will look into the source code to find some hints about it.

Reply:
> I suggest to switch to the built-in Node.js test runner or mocha.
> 
> Jest is slow and does pretty bad things internally that features are not working as expected (e.g., `instanceof` being broken, it manipulates assertion libraries output and handling and more).

That sounds reasonable. However, as the current issue is not related to a problem with Jest, I have created the new issue #20 for this suggestion.

Reply:
> Regarding the types field, as this is always empty, I don't know what it is used for. I will look into the source code to find some hints about it.

I have found out from the Git history that @ce-bo has added this "types" field in src/commands/outputMetrics.ts to store the "namespace" coupling metric in the commit https://github.com/MaibornWolff/metric-gardener/commit/e134ae4c7be9455262dee4b4719671c1a1c08134
However, this whole coupling metric and the code to add it to the field has been removed in https://github.com/MaibornWolff/metric-gardener/commit/2becab6f8d454c8984d84f45c262eaeee8f9a1d5, while the field "types" still remains.
I guess that we should remove the now unused field "types" again?

Reply:
I suggest to move this conversation to an issue. It seems unrelated to what should be achieved here?

Adding a general test script seems good enough, even if it's not possible to execute it as it's broken.

Reply:
As it has turned out that this issue is not related to the test script, thats correct.
I have opened the issues #25 and #26 for further discussion.